zoomus/components/user.py

```diff
@@ -48,7 +48,7 @@ def create(self, **kwargs):
 
     def update(self, **kwargs):
         util.require_keys(kwargs, "id")
-        return self.patch_request("/users/{}".format(kwargs.get("id")), params=kwargs)
+        return self.patch_request("/users/{}".format(kwargs.get("id")), data=kwargs)
```

Comment:
to keep the kwargs flexibility I used 
```suggestion
        id = kwargs.get("id")
        del kwargs["id"]
        return self.patch_request("/users/{}".format(id), **kwargs)
```
not happy with the del kwags["id"] but did not find better solution